core/src/i18n/pl.yaml

```
@@ -52,16 +57,27 @@ cron-jobs:
   concurrency-policy:
     descriptions:
       allow: Zezwalaj na współbieżne wykonania.
-      replace: Brak jednoczesnych wykonań, zastąp poprzednie uruchomienie nowym.
-      skip: Brak jednoczesnych wykonań, pomiń nowy przebieg, jeśli poprzedni jeszcze się nie zakończył.
+      replace: Brak współbieżnych wykonań, zastąp poprzednie uruchomienie nowym.
+      skip: Brak współbieżnych wykonań, pomiń nowy przebieg, jeśli poprzedni jeszcze się nie zakończył.
     title: Polityka współbieżności
   last-job-execution: Ostatnie wykonanie zadania
   last-schedule-time: Data ostatniego wykonania
   not-scheduled-yet: Jeszcze nie uruchomiono
   schedule: Harmonogram
-  title: Cron Jobs
+  title: CronJoby
 custom-resource-definitions:
   title: Definicje zasobów niestandardowych
+daemon-sets:
+  images: Obrazy
+  node-selector: Selektory Węzłów
```

```suggestion
  node-selector: Selektory węzłów
```
We should be consistent with capitalization used for terminology of the same type.